packages/target-tools/src/telemetryProvider.js

```diff
@@ -79,7 +87,8 @@ export function TelemetryProvider(
     getEntries,
     clearEntries,
     hasEntries,
-    executeTelemetries
+    executeTelemetries,
+    getMode
```

Comment:
Let's remove `getMode` from the exports since it should only be used internally in TelemetryProvider.